ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/BenchmarkTest.kt

```diff
@@ -23,7 +23,7 @@ private val testData = mutableMapOf(
 )
 
 class BenchmarkTest : ClientLoader() {
-    private val size = 1024
+    private val size = 1 // 1024
```

Comment:
?

Reply:
it's not critical for tests, but native works much faster